src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -377,8 +377,21 @@ ReportClientSideStats(
 
   std::cout << "    Request count: " << stats.request_count << std::endl;
   if (stats.delayed_request_count != 0) {
-    std::cout << "    Delayed Request Count: " << stats.delayed_request_count
-              << std::endl;
+    std::stringstream delay_data{""};
+    delay_data << "    Delayed Request Count: " << stats.delayed_request_count
```

Comment:
Didn't we already print the delayed request count somewhere? I don't see that removed/moved here, which has me wondering if it will be printed twice(?)

Reply:
and, does a raw delayed request count provide helpful information?

Reply:
This line is just updating the way its displayed. I am putting everything into a stringstream before I print the final message. It will not print twice. 
As for the raw value, it is likely not as valuable as the new percentage I added but it was previously printed and we tend to shy away from removing data that we provided to users.

Reply:
ok, not sure how I didn't see that it was replacing the exact text I was concerned was still somewhere else in the code.

Reply:
Discussed offline. Decided to remove the raw number in favor of a threshold and percentage.